python/GafferSceneUI/TransformQueryUI.py

```diff
@@ -0,0 +1,129 @@
+	# place the `filter` nodule inside the frame.
+	"nodeGadget:minWidth", 12.0,
+
+    plugs = {
+
+        "scene" : [
+
+            "description",
+            """
+            The scene to query the transform for.
+            """,
+
+        ],
+
+        "location" : [
+
+			"description",
+			"""
+			The location within the scene to query the transform at.
+
+			> Note : If the location does not exist then the query will not be
+			> performed and all outputs will be set to their default values.
+			""",
+
+			"plugValueWidget:type", "GafferSceneUI.ScenePathPlugValueWidget",
+			"scenePathPlugValueWidget:scene", "scene",
+			"nodule:type", "",
+
+		],
+
+        "space" : [
+
+            "description",
+            """
+            The space to query the transform.
+            """,
+
+            "preset:Local", GafferScene.TransformQuery.Space.Local,
+            "preset:World", GafferScene.TransformQuery.Space.World,
+            "preset:Relative", GafferScene.TransformQuery.Space.Relative,
+
+            "plugValueWidget:type", "GafferUI.PresetsPlugValueWidget",
+        ],
+
+        "relativeLocation" : [
+
+			"description",
+			"""
+			The location within the scene to query the transform for relative space mode.
+
+			> Note : If the location does not exist then the query will not be
+			> performed and all outputs will be set to their default values.
+			""",
+
+			"plugValueWidget:type", "GafferSceneUI.ScenePathPlugValueWidget",
+			"scenePathPlugValueWidget:scene", "scene",
+			"nodule:type", "",
+
+		],
+
+        "invert" : [
+
+            "description",
+            """
+            Invert the result transform.
+            """,
+
+        ],
+
+        "rotateOrder" : [
+
+            "description",
+            """
+            The order used for rotation query result.
+            """,
+
+            "preset:XYZ", GafferScene.TransformQuery.RotateOrder.XYZ,
+            "preset:XZY", GafferScene.TransformQuery.RotateOrder.XZY,
+            "preset:YZX", GafferScene.TransformQuery.RotateOrder.YZX,
+            "preset:YXZ", GafferScene.TransformQuery.RotateOrder.YXZ,
+            "preset:ZXY", GafferScene.TransformQuery.RotateOrder.ZXY,
+            "preset:ZYX", GafferScene.TransformQuery.RotateOrder.ZYX,
+
+            "plugValueWidget:type", "GafferUI.PresetsPlugValueWidget",
+        ],
+
+        "outMatrix" : [
+
+            "description",
+            """
+            4x4 matrix of the requested transform.
+            """,
+        ],
+
+        "outTranslate" : [
+            "description",
+            """
+            Translation component of requested transform.
+            """,
+        ],
+
+        "outRotate" : [
+            "description",
+            """
+            Rotation component of requested transform (degrees).
+            """
+        ],
+
+        "outScale" : [
+            "description",
+            """
+            Scaling component of requested transform.
+            """
+        ],
```

Comment:
Might it be worth putting these in an "Output" section like we do for the `FilterQuery` outputs?